Node cmake builds are failing

Without any changes, we suddenly see

```
-- NodeCMAKE: /home/travis/build/mapbox/directions-transit/node_modules/node-cmake/
-- Downloading: https://nodejs.org/download/release/v6.10.2/SHASUMS256.txt
-- NodeJS: Using node, version v6.10.2
-- Downloading: https://nodejs.org/download/release/v6.10.2/node-v6.10.2-headers.tar.gz
CMake Error at NodeJS.cmake:609 (set_target_properties):
  set_target_properties called with incorrect number of arguments.
Call Stack (most recent call first):
  src/bindings/CMakeLists.txt:16 (add_nodejs_module)
-- Node.js include path: 
-- Node.js libraries: 
-- Configuring incomplete, errors occurred!
```

on builds producing node bindings. Local version is still fine.

--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
     "node-pre-gyp": "^0.6.26"
   },
   "devDependencies": {
-    "node-cmake": "^2.2.1"
+    "node-cmake": "2.2.1"
   },
   "repository": "https://github.com/mapbox/directions-transit"
 }

--- a/src/bindings/CMakeLists.txt
+++ b/src/bindings/CMakeLists.txt
@@ -6,7 +6,8 @@ project(transit VERSION ${TRANSIT_VERSION_MAJOR}.${TRANSIT_VERSION_MINOR}.${TRAN
 
 message(STATUS "NodeCMAKE: ${CMAKE_SOURCE_DIR}/node_modules/node-cmake/")
 
-set(NodeJS_CXX_STANDARD 14 CACHE INTERNAL "Use C++14" FORCE)
+message(STATUS "LINK_FLAGS: ${LINK_FLAGS}")
+
 set(NodeJS_DOWNLOAD ON CACHE INTERNAL "Download node.js sources" FORCE)
 set(NodeJS_USE_CLANG_STDLIB OFF CACHE BOOL "Don't use libc++ by default" FORCE)
 include(${CMAKE_SOURCE_DIR}/NodeJS.cmake)
@@ -15,8 +16,6 @@ include(${CMAKE_SOURCE_DIR}/NodeJS.cmake)
 nodejs_init()
 add_nodejs_module(${PROJECT_NAME}
 	${binding_SOURCES})
-
-set_target_properties(${PROJECT_NAME} PROPERTIES CXX_STANDARD 14)
 
 target_include_directories(${PROJECT_NAME} SYSTEM PUBLIC "src"
     ${Boost_INCLUDE_DIRS}
@@ -33,6 +32,4 @@ target_link_libraries(${PROJECT_NAME}
     transitservice
     transittool)
 
-
-MESSAGE(STATUS "Node.js include path: ${NodeJS_INCLUDE_PATHS}")
-MESSAGE(STATUS "Node.js libraries: ${NodeJS_LIBRARIES}")
+set_target_properties(${PROJECT_NAME} PROPERTIES CXX_STANDARD 14)

